### ISSUE TITLE
Assign task to user

### ISSUE THREAD

**Issue by jcdude** [treeio/treeio#159]
I have just installed the latest treeio on centos 7. The problem i have is when I edit the task there is no way to assign it to a user. On the utube clip it shows a assign textbox but on the latest treeio there is no assign textbox and I am unable to assign any tasks

**Comment by tovmeod** [comment]
Hi, thanks for reporting the bug, note that the video tutorials are some years old so the interface may have changed slightly, in any case the user should be able to assign tasks, I will take a look at this and get back to you.

from what branch did you install? from the 2.0?

**Comment by jcdude** [comment]
That is correct I did a git clone of https://github.com/treeio/treeio.git and installed it straight off there

**Comment by jcdude** [comment]
I go to edit task and there is no assign option. Maybe you could send me a screen shot of where it is on your screen
